de.metas.swat/de.metas.swat.base/src/main/sql/postgresql/ddl/functions/update_TRL_Tables_On_AD_Element_TRL_Update.sql

```diff
@@ -1,19 +1,12 @@
+-- Function: public.update_trl_tables_on_ad_element_trl_update(numeric, character varying)
 
+-- DROP FUNCTION public.update_trl_tables_on_ad_element_trl_update(numeric, character varying);
 
-/*
-	When the AD_Element_trl has one of its values changed, the change shall also propagate to the linked table entries:
- 
-	AD_Column_TRL -- name, isTranslated
-	AD_Process_Para_TRL -- name, description, help, isTranslated
-	AD_Field_TRL --name, description, help, isTranslated
-	AD_PrintFormatItem_TRL -- printname, isTranslated
-*/
```

Comment:
i find this documentation useful :-)
wdyt about adding it to the function's comment (i.e. `comment on function..is ' ..';`) ?